tests/test_export.py

```diff
@@ -103,6 +103,8 @@ def setUpClass(cls):
 
         assert Feed.objects.count() == 3
         assert Post.objects.count() == 3
+        cls.root_dir = "/tmp"
+        cls.corpus_dir = "/tmp/corpus"
```

Comment:
Et voila. 

I got errors about lambdas when I handled it in the  setUp and tearDown, so I cleaned up the corpus directory on tearDown, and root_dir in class teardown.